src/main/interface/cli.c

```
@@ -936,6 +977,21 @@ static void cliSerialPassthrough(char *cmdline)
             if (strstr(tok, "tx") || strstr(tok, "TX"))
                 mode |= MODE_TX;
             break;
+        case 3:
+            // When programming Arduino based devices such as MinimOSD, the DTR line is used to control
+            // the reset. This parameter is the pin name, for example C8. This allows
+            // any GPIO to be used for driving DTR.
+            if (strToPin(tok, &serialPassthroughDtrTag)) {
```

Invalid pin format will be silently ignored